extensions/coffee/src/caffeinate.ts

```diff
@@ -2,11 +2,13 @@ import { showHUD } from "@raycast/api";
 import { runAppleScript } from "run-applescript";
 
 const Caffeinate = async (args?: string) => {
+  const validStringCaffeinateArgs = typeof args === "string";
+
   try {
     await runAppleScript(`do shell script "pgrep caffeinate"`);
     await showHUD("Your Mac is already caffeinated");
   } catch (_) {
-    runAppleScript(`do shell script "caffeinate -di${args ? ` ${args}` : ""}"`);
+    runAppleScript(`do shell script "caffeinate -di${validStringCaffeinateArgs ? ` ${args}` : ""}"`);
```

Comment:
just a nit: `${typeof args === "string" ? ...}` would save one line:)

Reply:
should be on PR now 😅 that's the hardest _simple_ fix, I've ever done heh.

Reply:
if you want, you could also add yourself to the contributors in the package.json with that one liner :) @GastroGeek

Reply:
why not, eh? 😄 done! thanks!